pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 streamdeck = "^0.9.3"
-pillow = "^9.4.0"
+pillow = "^10.0.0"
```

Comment:
pillow 10 brings breaking changes to the code of image_filter that need to be consider and need to be handle. could you please remove this update, the rest of changes LGTM

Reply:
All set. Thanks for the review @rkmax !